13/umbraco-commerce/release-notes/README.md

```diff
@@ -17,6 +17,18 @@ If you are upgrading to a new major version, check the breaking changes in the [
 
 This section contains the release notes for Umbraco Commerce 13 including all changes for this version.
 
+#### [13.1.5](https://github.com/umbraco/Umbraco.Commerce.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F13.1.5) (July 3rd 2024)
+
+* Added new `IRoundingService` to allow overriding the default rounding behaviour [#506](https://github.com/umbraco/Umbraco.Commerce.Issues/issues/506).
+* Added pesimistic locking to payment provider callback endpoint to prevent concurrency issues if the endpoint is called too multiple times at once [#533](https://github.com/umbraco/Umbraco.Commerce.Issues/issues/533).
```

Comment:
```suggestion
* Added pessimistic locking to the payment provider callback endpoint to prevent concurrency issues if the endpoint is called too many times at once [#533](https://github.com/umbraco/Umbraco.Commerce.Issues/issues/533).
```